.travis.yml

```diff
@@ -3,6 +3,12 @@ node_js:
 - '0.12'
 - '0.11'
 - '0.10'
+env:
+  global:
+    - secure: iaRFUMPT3/tYf4FZhDmWScwfnRCzk8AakeqDoiNhCx/z0O6LMha8gPuJkWF1Cnl34LmCkmQa1ZeFYht50Fjst0g7SEgjC4H4GOSLTuG1bJU3fqQYIxFrODJd0Zw0RpgrwE+I1sShwm67Ywf3PAJQWsqOQAACZjZnohjbsEElpJg=
```

Comment:
they are two env variables encrypted using travis key. they're provided by saucelabs, copied and pasted here.